### PR TITLE
Merging Development branch into Main

### PR DESCRIPTION
**Pull request by UmeshJeya**
- for peer review process.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -5,9 +5,25 @@ body {
...
+  font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  color: green;
+  text-transform: uppercase;
+  text-decoration: underline;
+  margin-bottom: 80px;
+  margin-top: 50px;
+  font-size: 52px;
 }
+
+
```
Comment (KylMoore) [inline src/index.css:21]: maybe add some media queries for the larger screens, the smaller screen actually doesn't look to bad below 640px. also add some padding for in between memes to make it clearer to see them!

File: package.json [hunk 2]
```diff
@@ -9,7 +9,9 @@
     "axios": "^1.3.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.7.1",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.6",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```
Comment (KylMoore) [inline package.json:17]: add a title to your index.html file! and add a favicon! https://formito.com/tools/favicon makes that super easy

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,98 @@
...
+    );  
+}
+  
+  return (
+  <div className="App">
+      <Fragment>
+      <Heading>Meme Generator!</Heading> 
+      <Content />
+      <GoTopButton />
+      </Fragment>
```
Comment (KylMoore) [inline src/App.js:43]: good use of Components

File: src/App.css [hunk 4]
```diff
@@ -1,7 +1,17 @@
 .App {
   text-align: center;
+  margin-right: 45px;
 }
```
Comment (rselvasoyak) [inline src/App.css:23]: Import ones: 
- Accessibility! hover and focus state for anything likable 
- flex- basis and flex for images to which when app mounts appear.
- Using border and background color with contrast to font-color to help them pop a bit more
- for individual meme page border for the main content,  using spacing, flexbox and maybe storing them in a container to visually indicate that they are part of same form and content. 
- Maybe using some text-shadow can help with a nice visual effect. 
- A lighter blue background can be easier on the eye. 
- using wrapper and setup snippet. 
- Border Radius can be very helpful

**Pull request merged**